install-dev-dependencies.sh

```diff
@@ -37,7 +38,7 @@ if type apt-get >/dev/null ; then
   $SUDO_SHIM apt-get install -y libxml2-dev libxml2-utils libxslt1-dev \
     python3.6-dev $BROWSERS zip sqlite3 python3-pip libcurl4-openssl-dev xvfb \
     libssl-dev git curl $CHROMEDRIVER
-  if ! type geckodriver >/dev/null; then
+  if ! type geckodriver >/dev/null 2>&1;  then
     curl -LO "https://github.com/mozilla/geckodriver/releases/download/v0.17.0/geckodriver-v0.17.0-linux64.tar.gz"
```

Comment:
For the `chromedriver` we detect the `$ARCH`. Would be great if we are consistent.